### PR TITLE
Integrate gutenberg-mobile release 1.60.1

### PR DESCRIPTION
**Pull request by geriux**
## Description
This PR incorporates the 1.60.1 release of gutenberg-mobile.
For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3872

Release Submission Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ buildscript {
     ext.wordPressUtilsVersion = 'develop-f98c841225fbc61031f32426aaef4e347b34f717'
     ext.wordPressLoginVersion = '0.0.4'
     ext.detektVersion = '1.15.0'
-    ext.gutenbergMobileVersion = 'v1.60.0'
+    ext.gutenbergMobileVersion = '3872-d59918b9aadcf958ce7b8a98ee636d571e58773f'
```
Comment (dcalhoun) [inline build.gradle]: Friendly reminder that we will need to update this to the GB Mobile tag after we create it.

**Review by dcalhoun — APPROVED** [review]
LGTM! Thank you!

File: build.gradle (hunk 1 above)
Comment (geriux) [reply build.gradle]: Thank you! Now that the release and tag are created I updated this 👍

**Comment by AliSoftware** [comment]
This has now been included as part of the new WPAndroid & JPAndroid `18.1-rc-2` betas 🎉

**Pull request merged**